src/LaunchDarkly.EventSource/Resources.resx

```diff
@@ -1,153 +1,159 @@
-              <xsd:attribute ref="xml:space" />
-            </xsd:complexType>
-          </xsd:element>
-          <xsd:element name="resheader">
-            <xsd:complexType>
-              <xsd:sequence>
-                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
-              </xsd:sequence>
-              <xsd:attribute name="name" type="xsd:string" use="required" />
-            </xsd:complexType>
-          </xsd:element>
-        </xsd:choice>
-      </xsd:complexType>
-    </xsd:element>
-  </xsd:schema>
-  <resheader name="resmimetype">
-    <value>text/microsoft-resx</value>
-  </resheader>
-  <resheader name="version">
-    <value>2.0</value>
-  </resheader>
-  <resheader name="reader">
-    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
-  </resheader>
-  <resheader name="writer">
-    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
-  </resheader>
-  <data name="Configuration_RetryDuration_Exceeded" xml:space="preserve">
-    <value>The maximum retry duration is {0}.</value>
-  </data>
-  <data name="Configuration_Value_Greater_Than_Zero" xml:space="preserve">
-    <value>Value must be greater than zero.</value>
-  </data>
-  <data name="EventSourceService_Read_Timeout" xml:space="preserve">
-    <value>A timeout occurred while reading the request from the remote EventSource API</value>
-  </data>
-  <data name="EventSource_204_Response" xml:space="preserve">
-    <value>Remote EventSource API returned Http Status Code 204.</value>
-  </data>
-  <data name="EventSource_Already_Started" xml:space="preserve">
-    <value>Invalid attempt to call Start() while the connection is {0}.</value>
-  </data>
-  <data name="EventSource_HttpResponse_Not_Successful" xml:space="preserve">
-    <value>Http Status Code '{0}' indicates an unsuccessful response returned from the remote EventSource API.</value>
-  </data>
-  <data name="EventSource_Invalid_MediaType" xml:space="preserve">
-    <value>HTTP Content-Type returned from the remote EventSource API does not match 'text/event-stream'.</value>
-  </data>
-  <data name="EventSource_Logger_Closed" xml:space="preserve">
-    <value>EventSource.Close called</value>
-  </data>
-  <data name="EventSource_Logger_Connection_Error" xml:space="preserve">
-    <value>Encountered exception in EventSourceService.ConnectToEventSourceApi method. Exception Message: {0}</value>
-  </data>
-  <data name="EventSource_Logger_Disconnected" xml:space="preserve">
-    <value>EventSource Disconnected. Automatically delaying {0}ms before reconnecting.</value>
-  </data>
-  <data name="EventSource_Response_Content_Empty" xml:space="preserve">
-    <value>Http Response contained no content.</value>
-  </data>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Please don't check in changes that touch the entire file when it's only because your IDE decided to reformat them. That makes it impossible to track what was really changed in the git history.